a3p-integration/proposals/README.md

```diff
@@ -78,15 +78,21 @@ the repo. The PR's CI will test it and once the PR is merged it will update the
 
 ## How to revise this directory after 
 
-Once `latest` has changed, if it was a Software Upgrade Proposal then the
-upgrade handler in master will fail.  If that poses a problem, you can set
-a3p-integration `agoricSyntheticChain.fromTag` to a specific version instead
-of `latest`. See https://ghcr.io/agoric/agoric-3-proposals for the available
-tags.
+Once a new proposal is merged into agoric-3-proposals, take the proposal name
+(the part after the colon) and make it the fromTag value in a3p-integration's
+package.json.
```

Comment:
> make it the fromTag value

This might be a little confusing since the tag would actually be formed as `use-${proposalName}`